packages/event-bus-default/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@medusajs/event-bus-default",
```

Comment:
question: why not just `@medusajs/event-bus`

Reply:
I'd like us to indicate in the name, that it is not for production. `-default` might not be the right suffix, but I believe `@medusajs/event-bus` might confuse people. Will think a bit about it

Reply:
Maybe `-simple`?

Reply:
-local ?